config/routes.rb

```diff
@@ -0,0 +1,60 @@
+Rails.application.routes.draw do
+  get '/v1/carriers/' => 'carriers#calculate'
+  get '/v1/carriers/selected' => 'carriers#selected'
+  get '/v1/tracking' => 'carriers#tracking'
```

Comment:
Hmm I think we were going to use that route and then never did, so should have deleted that one!